pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -263,7 +267,13 @@ func (policy *regoEnforcer) EnforceDeviceMountPolicy(target string, deviceHash s
 		"target":     target,
 		"deviceHash": deviceHash,
 	}
-	allowed, err := policy.allowed("mount_device", input)
+
+	results, err := policy.query("mount_device", input)
```

Comment:
can't `query` call into `allowed` and return the `results` when appropriate? this way we don't need to `query` and then `allowed`?

Reply:
Query isn't used just for enforcement point querying. It is also used for querying the "reason" rule, to find out if there is additional error information. Also, we need the results later to update the metadata. I tried a few different configurations, and this one was cleanest.